### ISSUE TITLE
Model 3.0.0 pulls in spacy 2.x

### ISSUE THREAD

**Issue by h-vetinari** [conda-forge/spacy-model-en_core_web-feedstock#11]
In the `allennlp-semparse` [feedstock](https://github.com/conda-forge/allennlp-semparse-feedstock/pull/2), there's a dependence on _only_, but despite the pinning in the recipe, this ends up pulling in an old spacy:
```
    spacy:                      2.3.5-py37h6dcda5c_1         conda-forge
    spacy-model-en_core_web_sm: 3.0.0-pyh44b312d_0           conda-forge
```

**Comment by bollwyvl** [comment]
uhoh! will take a look this weekend.

**Comment by setu4993** [comment]
This should be fixed now with #15 where we added a run time constraint to match the version of the model. Closing...
